src/Utils/Helpers.spec.ts

```diff
@@ -93,41 +94,106 @@ describe('Helpers', () => {
   });
 
   describe('getExecArgs', () => {
-    it('should return current exec args', () => {
-      const result = Helpers.getExecArgs();
+    describe('unescaped', () => {
+      it('should return current exec args', () => {
+        // arrange & act
+        const result = Helpers.getExecArgs({
+          escape: false
+        });
```

Comment:
Split "arrange" and "act" phrases when it is possible
```suggestion
        // arrange 
        const input = {
          escape: false
        };
        
        // act
        const result = Helpers.getExecArgs(input);
```